### ISSUE TITLE
HW GPU PICA errors 

### ISSUE THREAD

**Issue by newuser9290** [Lime3DS/Lime-3DS-Emulator#486]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Latest build 

### Description of Issue

Mainly with Mario Golf lot of hw GPU pica shader issues. Having SPIRV shaders off doesn't not remedy. Game is playable but will kick me out if it stutters enough tested on stock GPU and multiple latest drivers from K1MCH1 and weab 

### Expected Behavior

Should play normal 

### Reproduction Steps

Load Mario Golf 

### Log File

[lime3ds_log.old.txt](https://github.com/user-attachments/files/17249806/lime3ds_log.old.txt)

Sorry for the long ass log, towards the bottom are the pica issues 

### System Configuration

CPU: Snapdragon 8 Gen 3 
GPU/Driver: Adreno w/Latest Turnip and stock GPU 
RAM:12
OS:Android 14

**Comment by rtiangha** [comment]
Does it also happen on PabloMK7's fork? I'm curious because he's got some PICA commits that haven't made its way over here.

**Comment by newuser9290** [comment]
> Does it also happen on PabloMK7's fork? I'm curious because he's got some PICA commits that haven't made its way over here.

Yeah, I tested it on his latest action build and still the same issue. It runs fine when it works, its the PICA issues that cause a major slowdown. If I increase CPU JIT to around 140 it speeds up the process but then I have to revert back

Edit: Also Pablos latest action build talking about leaked memory addresses the ncch errors in the log as well, not sure if the Lime3DS team wants to look at that as well

**Comment by newuser9290** [comment]
![Screenshot_20241011_024406_Lime3DS.jpg](https://github.com/user-attachments/assets/8e6af27a-244c-4317-8a42-d21d7b3442e1)

Same pica errors in the above log with more issues. Screenshot posted

**Comment by newuser9290** [comment]
![Screenshot_20241011_023328_Lime3DS.jpg](https://github.com/user-attachments/assets/870b0edf-0b50-4d92-97c5-759fa110b2b0)

**Comment by newuser9290** [comment]
> Does it also happen on PabloMK7's fork? I'm curious because he's got some PICA commits that haven't made its way over here.

Don't mean to bother a couple weeks later. I redumped my games in cia format, and when I boot Mario Golf via HOME any and all stutters are gone. When you boot via Lime3DS or Citra menu, the stutters happen. Idk why but figured you'd find it interesting
